OTDefaultAudioDevice.h

```diff
@@ -7,33 +7,49 @@
 #import <Foundation/Foundation.h>
 #import <OpenTok/OpenTok.h>
 
-#define kMixerInputBusCount 2
-#define kOutputBus 0
-#define kInputBus 1
-
 #define AUDIO_DEVICE_HEADSET     @"AudioSessionManagerDevice_Headset"
 #define AUDIO_DEVICE_BLUETOOTH   @"AudioSessionManagerDevice_Bluetooth"
 #define AUDIO_DEVICE_SPEAKER     @"AudioSessionManagerDevice_Speaker"
 
 @interface OTDefaultAudioDevice : NSObject <OTAudioDevice>
-{
-  AudioStreamBasicDescription stream_format;
-}
 
 /**
- * Audio device lifecycle should live for the duration of the process, and
- * needs to be set before OTSession is initialized.
- *
- * It is not recommended to initialize unique audio device instances.
```

Comment:
:shrug: material, my only opinion is that a `sharedInstance` of a resource (audio or video) is not necessarily needed for our use case, as we don't overlap the tests, and hopefully someone will practice during ~10 minutes between them